framework/test/mocha/functional/http/get/accounts/accounts.js

```diff
@@ -303,14 +303,20 @@ describe('GET /accounts', () => {
 
 			it('using no sort return accounts sorted by both balance and address in asending order as default behavior', async () => {
 				return accountsEndpoint.makeRequest({}, 200).then(res => {
-					const balances = _.clone(res.body.data);
+					const balances = _.cloneDeep(res.body.data);
 					expect(
 						balances.sort((a, b) => {
 							const aBignumBalance = new Bignum(a.balance);
-							return (
-								aBignumBalance.minus(b.balance) ||
-								a.address.localeCompare(b.address)
-							);
+
```

Comment:
maybe something like
```
if (aBignumBalance.gt(b.balance)) {
  return 1;
}
if (aBignumBalance.lt(b.balance)) {
  return -1;
}

return a.address.localeCompare(b.address);
```

Reply:
You need to combine both the balance and address comparison, I think you are only sorting one item otherwise.

Reply:
In above, it's sorting by balance, and if the balance is the same, sort by address?